apps/api/src/form/form.service.ts

```diff
@@ -24,15 +26,22 @@ export class FormService {
       confirmationId,
     } as Partial<FormEntity>);
 
+    if (!newForm.version) {
+      newForm.version = '1.0.0';
```

Comment:
Should we default assume un-specified submissions are 1.0.0? This also doesn't fit the format I've seen previously, I thought the value was `v1`

Reply:
Again this is defensive code when validation is not present